frontend/src/views/main/project/Debugger.vue

```diff
@@ -144,6 +144,7 @@ async function showPastSessions() {
     params: {
       projectId: props.projectId.toString()
     }
+  }).catch(() => {
```

Comment:
I'm not a big fan of catching error and silencing them. Apparently a solution to update query params on the same route is to push the params only, but we will need to check if the route is the same or not each time...

```
await router.push({
    params: {
      projectId: props.projectId.toString()
    }
  })
```